File: CSharp-Part-2/6. Strings and Text Processing/StringsAndTextProcessing/ReverseString/ReverseString.cs
// Write a program that reads a string, reverses it and prints the result at the console.

using System;
using System.Text;

class ReverseString
{
    static void Main()
    {
        string inputStr = Console.ReadLine();
        Console.WriteLine(Reverse(inputStr));
    }

    static string Reverse(string str)
    {
        StringBuilder finalStr = new StringBuilder();
        for (int i = str.Length - 1; i >= 0; i--)
        {
            finalStr.Append(str[i]);
        }
        return finalStr.ToString();
    }
}

File: JavaScript-Fundamentals/11. Strings/reverse-string.js
// Write a JavaScript function that reverses a string and returns it.

function reverse(str) {
	var i,
		len = str.length,
		result = '';

	for (i = str.length - 1; i >= 0; i -= 1) {
		result += str[i];
	}

	return result;
}

console.log(reverse('sample'));